collectors/python.d.plugin/sensors/metadata.yaml

```
@@ -27,8 +27,8 @@ meta:
 overview:
   data_collection:
     metrics_description: >
-      Examine Linux Sensors metrics with Netdata for insights into hardware health and performance. 
-
+      Examine Linux Sensors metrics with Netdata for insights into hardware health and performance.
+      
       Enhance your system's reliability with real-time hardware health insights.
```

I think this is the correct thingy that will make yamllint happy
```suggestion
      Examine Linux Sensors metrics with Netdata for insights into hardware health and performance.

      Enhance your system's reliability with real-time hardware health insights.
```